src/components/protocol_handler/src/protocol_handler_impl.cc

```diff
@@ -1121,7 +1326,26 @@ RESULT_CODE ProtocolHandlerImpl::HandleControlMessageStartSession(
                           PROTECTION_OFF);
       return RESULT_OK;
     }
-    if (ssl_context->IsInitCompleted()) {
+    ProtocolPacket::ProtocolVersion* fullVersion;
+    std::vector<std::string> rejectedParams(0, std::string(""));
+    if (packet.data_size() != 0) {
```

Comment:
Can you add a comment saying a control packet with data must be at least protocol version 5?